### PR TITLE
add 2 files hw2

### PR DESCRIPTION
**Pull request by AnastasiaPletser**

### CODE REVIEW

File: students/AnastasiaPletser/hw-1/index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+  <p>I'm just a text</p>  
+  <script> alert('One option') </script>
+ <script src="./index.js"> </script> 
+</body>
+</html>
```
Comment (Temu4) [inline students/AnastasiaPletser/hw-1/index.html:14]: Please add a new line at the bottom of your files.
Why? [link](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)
How to? [link](https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files)

File: students/AnastasiaPletser/hw-1/index.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+  <p>I'm just a text</p>  
+  <script> alert('One option') </script>
+ <script src="./index.js"> </script>
```
Comment (Temu4) [inline students/AnastasiaPletser/hw-1/index.html:12]: You also can use auto-formatter here. It'll make your codding process more easer =)

File: students/AnastasiaPletser/hw-1/index.js [hunk 3]
```diff
@@ -0,0 +1 @@
+alert('Second option')
```
Comment (Temu4) [inline students/AnastasiaPletser/hw-1/index.js:1]: Please add a new line at the bottom of your files.
Why? [link](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)
How to? [link](https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files)

File: students/AnastasiaPletser/hw-2/answers_2.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+typeof (null === '');//boolean (since the comparison)
+typeof (!! 'false' === !! 'true');  //boolean (since the comparison)
+typeof (['x'] =='x'); //boolean (since the comparison)
+typeof !! {}; //boolean (!! Negation is of type boolean)
+typeof ![]; //boolean (! Negation is of type boolean)
+typeof([] + null + 1); // string (object[]+value=string)
+typeof ([1, 2, 3] === [1, 2, 3]); //boolean (since the comparison)
+typeof ({}+ [] + {} + [1]); // string (The sum of the object is aqual to the string type)
+typeof (!+[] + [] + ![]); // string ((The sum of the object is aqual to the string type))
+typeof (new Data(0) - 0); // object (All constructor functions created with 'new' will be of type 'object')
```
Comment (Temu4) [inline students/AnastasiaPletser/hw-2/answers_2.md:19]: `object` isn't a primitive type, so it couldn't be as a result of any logical/math operations...

File: students/AnastasiaPletser/hw-2/answers_2.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+typeof (new Data(0) + 0);  // object (All constructor functions created with 'new' will be of type 'object')
```
Comment (Temu4) [inline students/AnastasiaPletser/hw-2/answers_2.md:20]: the same

**Review by Temu4 — CHANGES_REQUESTED** [review]

**Comment by Temu4** [comment]
@AnastasiaPletser  hey, do you need some help with this PR?

File: students/AnastasiaPletser/hw-2/answers_2.md (hunk 4 above)
Comment (AnastasiaPletser) [reply students/AnastasiaPletser/hw-2/answers_2.md:19]: typeof (new Data(0) - 0)    //number (result =0)

File: students/AnastasiaPletser/hw-2/answers_2.md (hunk 5 above)
Comment (AnastasiaPletser) [reply students/AnastasiaPletser/hw-2/answers_2.md:20]: typeof (new Data(0) + 0);   // string (Thu Jan 01 1970 03:00:00 GMT+0300)

**Pull request merged**